presto-hive/src/main/java/com/facebook/presto/hive/metastore/thrift/HiveMetastoreClient.java

```diff
@@ -112,4 +112,7 @@ boolean grantPrivileges(PrivilegeBag privilegeBag)
 
     boolean revokePrivileges(PrivilegeBag privilegeBag)
             throws TException;
+
+    List<String> setUGI(String userName, List<String> groupNames)
```

Comment:
please add some javadoc. `UGI` meaning is not obvious.

Reply:
thx, I will fix it